graph-builder/src/status.rs

```diff
@@ -64,3 +64,56 @@ pub fn serve_readiness(
     };
     Box::new(future::ok(resp))
 }
+
+#[cfg(test)]
+mod tests {
+    extern crate tokio;
+
+    use self::tokio::runtime::current_thread::Runtime;
+    use crate::graph::State;
+    use crate::status;
+    use actix_web::test::TestRequest;
+    use parking_lot::RwLock;
+    use std::collections::HashSet;
+    use std::sync::Arc;
+
+    fn common_init() -> Runtime {
```

Comment:
What do you think of using `Result` everywhere in the tests? This could then be 
```suggestion
    fn common_init() -> Result<Runtime, Box<Error>> {
```